README.md

```diff
@@ -544,7 +544,7 @@ To generate a migration for the content db:
 
 ```
 cd explore-education-statistics\src\GovUk.Education.ExploreEducationStatistics.Admin
-dotnet ef migrations add EES1234MigrationNameHere --context ContentDbContext --output-dir Migrations/ContentMigrations -v
+dotnet ef migrations add EES1234_MigrationNameHere --context ContentDbContext --output-dir Migrations/ContentMigrations -v
```

Comment:
I'm happy to leave it as is like this - I find them more readable and we've got a mix already, so you happy to continue on like this for the future?

Reply:
Yes I think that's fine and I'm going to join you and revert back to using the format `EES1234_MigrationNameHere` because I find it more readable as well.